blog/README.md

```diff
@@ -16,6 +16,15 @@ Note you may need to open the "Screen Options" dialogue at the top of the page t
 
 Similarly, don't forget the "Featured Image" field in the sidebar. If you set an image here, it will be used for social media. This image is not displayed in the post content, however, so if you want to use it as a big image at the top of your post, you'll have to do that AND set the featured image.
 
+## Categories and Tags
+
+Wordpress has a couple tools for organizing your posts: categories and tags. In the current interation of cloudfour.com, we primarily use categories, and rarely use tags. Here's some background context from Tyler:
+
+>We simplified to displaying categories as "Topics" on our front-end. We only use tags when we want to have some point of organization that isn't exposed to that. As an example, a tag is what's used to determine the featured articles on the home page.
+>
+>It's possible that in the future we'll want to revisit this. As our staff has grown, the variety of things we're writing about has broadened. But the short version for now is that categories are a public-facing thing, tags are sort of an internal tool we don't expose on the front-end.
```

Comment:
Boy, I can _feel_ how tired I am when I read this back! 😆 

I'm not sure my quote is so wise that it deserves to be recreated in full.

You might be able to simplify this section to not include a quote at all, but here's a better written one from me if you find that helpful:

> In past versions of the site, a large amount of tags proved tricky to navigate. So currently, we only expose categories (though we call them "Topics") in the front-end. Sometimes we use tags as an organizational tool behind the scenes, like flagging "featured" articles for the homepage.

Reply:
Thanks Tyler! I fixed the typo and swapped in your improved quote. 

If you'd like I can simplify to avoid the quote, but I think I'd likely just be rewriting the contents of your quote in my own words.